data/test/smallseq/Ref.h.vcf

```diff
@@ -0,0 +1,98 @@
+##ALT=<ID=db22dfc14799b1aa666eb7d571cf04ec,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=2,Start=16500,End=17499,Checksum=Md5,RefRange=db22dfc14799b1aa666eb7d571cf04ec>
+##ALT=<ID=ddbae40728836c4c245f9ccb7e651515,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=1,Start=44000,End=44999,Checksum=Md5,RefRange=ddbae40728836c4c245f9ccb7e651515>
+##ALT=<ID=e07d04d2fc96b0f8f075aa5641785f78,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=1,Start=16500,End=17499,Checksum=Md5,RefRange=e07d04d2fc96b0f8f075aa5641785f78>
+##ALT=<ID=e4f5f60483b68badd6caf4cc2812d2bc,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=2,Start=28500,End=32999,Checksum=Md5,RefRange=e4f5f60483b68badd6caf4cc2812d2bc>
+##ALT=<ID=f2a26406aaa6aefdc20ccdf8cb313dff,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=1,Start=12000,End=16499,Checksum=Md5,RefRange=f2a26406aaa6aefdc20ccdf8cb313dff>
+##ALT=<ID=f62948ca6d4385c880321727ba494bec,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=1,Start=50500,End=54999,Checksum=Md5,RefRange=f62948ca6d4385c880321727ba494bec>
+##ALT=<ID=fafafee7c250c76b7e0571fde286022e,Description="haplotype data for line: Ref",Number=6,Source="data/test/smallseq/Ref.fa",Contig=1,Start=49500,End=50499,Checksum=Md5,RefRange=fafafee7c250c76b7e0571fde286022e>
+##FORMAT=<ID=AD,Number=3,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth (only filtered reads used for calling)">
+##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="Genotype Quality">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=PL,Number=G,Type=Integer,Description="Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification">
+##INFO=<ID=AF,Number=3,Type=Integer,Description="Allele Frequency">
+##INFO=<ID=ASM_Chr,Number=1,Type=String,Description="Assembly chromosome">
+##INFO=<ID=ASM_End,Number=1,Type=Integer,Description="Assembly end position">
+##INFO=<ID=ASM_Start,Number=1,Type=Integer,Description="Assembly start position">
+##INFO=<ID=ASM_Strand,Number=1,Type=String,Description="Assembly strand">
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
+##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
+##INFO=<ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data">
+##contig=<ID=1,length=55000>
+##contig=<ID=2,length=55000>
+##reference=https://s3.amazonaws.com/maizegenetics/phg/phgV2Test/Ref.fa
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Ref
+1	0	.	G	<NON_REF>	.	.	END=999	GT:AD:DP	0:2,0:2
```

Comment:
The latest discussion was that the ref haplotype id should be the ALT and that GT should then be 1. Is that still the plan and if so, do you still want to merge this version?

Reply:
@pjbradbury The REF hvcf file will be updated when I have updates to BuildRefVCF coded, which I"m working on now.   I wanted to get the basic LoadVCF finished and committed as other parts of the pipeline need it.